app/src/angular/controllers/member/wallet/manage-token-controller.js

```diff
@@ -31,19 +31,24 @@ function ManageTokenController($rootScope, $scope, $state, $log, $mdDialog, $sta
     function prepareBalance() {
         if ($scope.symbol === 'ETH') {
             // ETHER
-            $scope.balance = CommonService.commasAfterNumber($rootScope.wallet.balanceEth, 2);
-            $scope.balanceUsd = CommonService.commasAfterNumber($rootScope.wallet.balanceInUsd, 2);
+            let ethRoundedBalanceUSD = $rootScope.wallet.balanceInUsd > 0 ? $rootScope.wallet.balanceInUsd.toFixed(2) : $rootScope.wallet.balanceInUsd;
```

Comment:
Why aren't we using the [built-in formatting functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) that do all this for us?

Reply:
we have a service in CommonService which puts commas after numbers we want to. we can use this built-in function instead that, but we still have to use toFixed method, because we want to show only two decimals rounded in the end, when its not whole number.

Reply:
>but we still have to use toFixed method, because we want to show only two decimals rounded in the end, when its not whole number.

No you don't. The built-in function automatically rounds to the correct number of digits as long as you specify `{ style: 'currency', currency: 'USD' }` in the options.